robotoff/app/api.py

```diff
@@ -1141,6 +1142,31 @@ def on_get(self, req: falcon.Request, resp: falcon.Response):
         resp.media = response
 
 
+class UnansweredQuestionCollection:
+    def on_get(self, req: falcon.Request, resp: falcon.Response):
+        response: JSONType = {}
+        page: int = req.get_param_as_int("page", min_value=1, default=1)
+        count: int = req.get_param_as_int("count", min_value=1, default=25)
+        question_type: str = req.get_param("question_type")
+
+        insights = list(
+            get_insights(
+                keep_types=[question_type],
+                group_by=True,
```

Comment:
I don't think it's a good option to have a boolean there. 

So either we put a string, the field we want to group on: here `group_by=Insight.value_tag` (which as the draw back of using Insight)

or we call the parameter group_by_value_tag (which is a bit less elegant but may better do the job here).